dev-java/abcl/abcl-1.8.0.recipe

```diff
@@ -0,0 +1,51 @@
+SUMMARY="Armed Bear Common Lisp (ABCL) is a Java-based common lisp implementation"
```

Comment:
Maybe:
```diff
- SUMMARY="Armed Bear Common Lisp (ABCL) is a Java-based common lisp implementation"
+ SUMMARY="A Java-based common lisp implementation"
```
See: https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#summary

Reply:
Thank you for the reminder, I have updated the summary